### DISCUSSION TITLE
adding count-down before taking a  screenshot

### DISCUSSION THREAD

**Discussion by bulldozerdev** [flameshot-org/flameshot#2756] [Ideas]
Could you please add the count-down feature before taking the screenshot .

**Comment by borgmanJeremy** [comment]
Are you asking for a visual countdown or a way to add a delay?

**Comment by bulldozerdev** [comment]
Hello,

Im asking for visual count down.

Thank you

**Comment by bulldozerdev** [comment]
Hello,
Count-down feature not added yet, could you please add it ASAP. 
When i click on take a screenshot i need to see a count down then after 5 second i can grab an area to take the screenshot.

Regards, Hamza

**Reply by mmahmoudian** [reply]
So you want to see a countdown that shows you how many seconds is left from the delay until the screenshot is taken? Kinda like [in movies](https://www.youtube.com/watch?v=u7gf6_85-jQ)?

**Reply by borgmanJeremy** [reply]
You just suggested this a few days ago on an issue. Please don't spam us asking to get it done fast.


First we need to decide if we want to add it. Then we need to decide if it's feasible.

At that point it will get added when someone feels like working on it. Feel free to sponsor a developer on rysolv to add the feature if it is that important AFTER the decision is made to add such a feature.
